#!/usr/bin/env python2

PACKAGE = "cpe_project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add( "upper_first" , int_t , 1 , "first" , 0 , 0 , 255 )
gen.add( "upper_second" , int_t , 1 , "second" , 0 , 0 , 255 )
gen.add( "upper_third" , int_t , 1 , "third" , 0 , 0 , 255 )
gen.add( "down_first" , int_t , 1 , "first" , 0 , 0 , 255 )
gen.add( "down_second" , int_t , 1 , "second" , 0 , 0 , 255 )
gen.add( "down_third" , int_t , 1 , "third" , 0 , 0 , 255 )

exit( gen.generate( PACKAGE , "project_configure" , "Simple3Data" ) )
